data_architect_misc/data_transformer/transform_functions/latam_mexico_transform_functions.py

```diff
@@ -0,0 +1,92 @@
+
+
+class LatamMexicoTransformFunctions(CommonCompHarmTransformFunctions):
+    """
+    All custom (uncommon) transform functions **SPECIFIC to
+    individual processing task** must be defined as part
+    of this class.
+    """
+    MEXICO_SPECIFIC_CATEGORY_MAPPINGS = {
+            "(?i).*APARATOS\sDOMESTICOS.*": "Other",
+            "(?i).*APARATOS.*ACCESORIOS\sDE\sVIDEO.*": "Other",
+            "(?i).*ALIMENTOS.*ARTICULOS.*": "Other",
+            "(?i).*ALIMENTOS.*INFANTILES.*": "Other",
+            "(?i).*ACCESORIO.*ARTICULO.*PERSONAL.*": "Other",
+            "(?i).*ACCESORIOS.*REFACCIONES\sVEHICULOS\sMOTOR.*": "Other",
+            "(?i).*ALIMENTOS.*ENLATADO.*INSTANTANE.*": "Other",
+            "(?i).*ARTE.*CUTURA.*": "Other",
+            "(?i).*AGRICULTURA.*": "Other",
+            "(?i).*APARATOS\sPARA\sLA\sSALUD.*": "Other",
+            "(?i).*BOTANAS.*": "Other",
+            "(?i).*BLANCOS.*": "Home Care",
+            "(?i).*CAFES/TES/MODIFICADOR\sDE\sLECHE.*": "Other",
+            "(?i).*COMPUTACION.*": "Other",
+            "(?i).*CHOCOLATES/DULCES/GOLOSINAS.*": "Other",
+            "(?i).*CONDIMENTOS/ADEREZOS.*": "Other",
+            "(?i).*CAMPINGS/CAMPAMENTOS.*": "Other",
+            "(?i).*COMBUSTIBLE.*LUBRICANTE.*": "Other",
+            "(?i).*DISCOS\sCASSETTES\sDISCOS\sCOMPACTOS.*": "Other",
+            "(?i).*EDUCACION.*": "Other",
+            "(?i).*EDITORAS\sDE\sLIBROS.*": "Other",
+            "(?i).*GASTRONOMIA/CENTROS\sNOCTURNOS.*": "Other",
+            "(?i).*GRANOS.*LEGUMINOSAS.*": "Other",
+            "(?i).*INSTRUMENTOS.*ACCESORIOS\sMUSICALES.*": "Other",
+            "(?i).*INSTITUCIONAL.*": "Other",
+            "(?i).*INMOBILIARIAS.*": "Other",
+            "(?i).*ILUMINACION/PILAS/FOCOS.*": "Other",
+            "(?i).*JUEGOS\sY\sJUGUETES.*": "Other",
+            "(?i).*MEDIOS\sDE\sCOMUNICACI.*": "Other",
+            "(?i).*MATERIAL.*CONSTRUCCION.*": "Other",
+            "(?i).*MAQUINA.*EQUIPO.*PARA\sESCUELA.*": "Other",
+            "(?i).*MAQUINAS.*REFACCIONES.*": "Other",
+            "(?i).*MENUDEO.*": "Other",
+            "(?i).*MOTOCICLETA.*": "Other",
+            "(?i).*NO\sDISPONIBLE.*": "Other",
+            "(?i).*PUBLICIDAD\sCOMPARTIDA.*": "Other",
+            "(?i).*PANIFICACION.*": "Other",
+            "(?i).*PRESTACION\sDE\sSERVICIO\sSOCIAL.*": "Other",
+            "(?i).*PRODUCTOS\sGRASOS.*": "Other",
+            "(?i).*PALETAS/HELADOS.*": "Other",
+            "(?i).*PRODUCTO.*DOMESTICO.*DESECHABLE.*": "Other",
+            "(?i).*RELOJ.*JOYA.*": "Other",
+            "(?i).*ROPA.*": "Other",
+            "(?i).*SEGURO.*": "Other",
+            "(?i).*TELEMERCADEO.*": "Other",
+            "(?i).*TURISMO.*VIAJE.*": "Other",
+            "(?i).*UTENSILLOS\sDOMESTICOS.*": "Other",
+            
+    }
+
+    def __init__(self, config):
+        self.config = config
+
+    def add_HARMONIZED_CATEGORY_column_using_existing_category_column_with_country_specific_mappings(
```

Comment:
We need to move this function to common_comp_harm_transform_functions.py. Similarly, please remove this function from apac_hong_kong_transform_functions.py, apac_malaysia_transform_functions.py, apac_taiwan..., apac_thailand..., latam_el_salvador..., latam_honduras..., latam_mexico...., latam_nicaragua....

After moving this function to common_comp_harm_transform.py and removing them from the above custom transform files, please test the code to make sure they are working as expected. (Just testing one or two countries should be enough).

Reply:
Update the comment for this function too. And say something like, "This function helps map the product/subcategory names to 1PH categories using product items found in a specific country (as opposed to the generic one that we can use across the globe/region.)"